.github/workflows/kubernetes-deploy.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
             --set gateway.application.limits.keepClusterOnComplete=false \
             .
           GATEWAY=$(kubectl get pod -l app.kubernetes.io/name=gateway -o name)
-          kubectl wait --for=condition=Ready "$GATEWAY" --timeout 7m
+          kubectl wait --for=condition=Ready "$GATEWAY" --timeout 9m
```

Comment:
this shouldn't be taking that long... we should look into why the pods aren't coming ready

Reply:
Mmm... how can we test it? I mean, it's not something that I found in this PR but in others too

Reply:
actually, seems the gateway is crashing:
```
gateway-554bf468d7-8qnlb            0/1     CrashLoopBackOff   5 (57s ago)   5m4s
```

Reply:
Weird, locally I was not noticing that it was crashing. I will take a look into it 👍

Reply:
definitely in local doesn't crash to me 😅 

do you have `kind` locally @psschwei ?

Reply:
I just checked -- the gateway pod also restarted for me locally (same issue with probes failing), but after the first restart things righted themselves (eventually, it took about 90s for the pod to become ready)